go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/hashicorp/go-hclog v1.2.1
 	github.com/hashicorp/go-retryablehttp v0.7.1
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.17.0
-	github.com/okta/okta-sdk-golang/v2 v2.13.0
+	github.com/okta/okta-sdk-golang/v2 v2.13.1-0.20220629214615-7167dfb447ff
```

Comment:
any reason this version has this sha(?) attached to it?

Reply:
It targets a specific version of okta-sdk-golang https://github.com/okta/okta-sdk-golang/pull/315